aten/src/ATen/native/cuda/EmbeddingBackwardKernel.cu

```diff
@@ -13,7 +13,8 @@
 
 #include <thrust/execution_policy.h>
 #include <thrust/unique.h>
-#include <thrust/device_vector.h>
+
+#include <torch/types.h>
```

Comment:
@madsbk  I think this file should not include headers under `torch/csrc/api`. If you need to use `torch::kFloat16` and `torch::kFloat32`, you can replace them with `at::kHalf` and `at::kFloat` [[link](https://github.com/pytorch/pytorch/blob/master/torch/csrc/api/include/torch/types.h#L24-L25)].